lib/membrane_rtc_engine/engine.ex

```diff
@@ -525,19 +594,82 @@ defmodule Membrane.RTC.Engine do
     {{:ok, tracks_msgs}, state}
   end
 
-  @spec publish(msg :: any()) :: Keyword.t()
+  @doc """
+  This function create a notification, which should be send to all others endpoints in room.
+  """
+  @spec publish(msg :: any()) :: [Membrane.Notification.t()]
   def publish(msg), do: [notify: {:publish, msg}]
 
-  @spec subscribe([{String.t(), atom()}]) :: Keyword.t()
+  @doc """
+  This function create a notification, which is response for {:new_tracks, tracks} message. It should contains a list
+  in form of pairs {track_id, track_format}, where track_id is id of track on which this endpoint want to subscribe and track_format
+  is the format in which endpoint want to receive stream (e.g. raw, RTP). All possible format should be sent in field format in track in
+  message {:new_tracks, tracks}.
+  """
```

Comment:
```suggestion
  @doc """
  Subscribes for tracks in given format.
  
  Endpoint  will be notified about track readiness in `Membrane.Bin.handle_pad_added` callback.
  `tracks` is a list in form of pairs `{track_id, track_format}`, where `track_id` is id of track this endpoint subscribes for 
  and `track_format` is the format of track that this endpoint is willing to receive. 
  If `track_format` is `:raw` Endpoint will receive track in `Track.encoding()` format. 
  
  This function returns list of `Membrane.Element.Action.notif_t()` that should be returned from Membrane callback.
  """
 ```
 
 Let's try to adjust remaining docs to be similiar to these two :wink: